src/diff/children.js

```diff
@@ -115,7 +115,8 @@ export function diffChildren(
 					(refs || (refs = [])).push(
```

Comment:
What's the size if call `applyRef` twice?
```js
if ((j = childVNode.ref) && oldVNode.ref != j) {
   if (!refs) refs = [];
   if (oldVNode.ref) refs.push(oldVNode.ref, null, childVNode);
   refs.push(j, childVNode._component || ... );
}
```
Or something similar...

The if statement in `diffChildren` already checks if the refs are different so ideally we could reuse that `if` and not duplicate it in `applyRef` (if I'm reading this code correctly).